mv3/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "ipfs-companion-mv3",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "esbuild": "^0.14.11",
+    "ipfs-http-client": "^53.0.1",
+    "is-fqdn": "^2.0.1",
+    "is-ipfs": "^6.0.2",
+    "lru-cache": "^6.0.0",
+    "rimraf": "^3.0.2",
+    "web-ext": "^6.6.0",
+    "webextension-polyfill": "^0.8.0"
+  },
+  "scripts": {
+    "lint": "eslint --ignore-path .eslintignore --ext .ts .",
+    "prebuild": "rimraf build",
+    "build": "esbuild ./addon/background.ts --bundle --outfile=./build/main.js",
+    "postbuild": "cp -r ./addon/* build/",
+    "build:firefox": "web-ext build --source-dir ./build/",
+    "format": "prettier --ignore-path ../.gitignore --write \"**/*.+(js|ts|json)\"",
+    "test": "echo 'testing ipfs companion mv3'",
+    "ci:install": "npx yarn@1.22.10 install --frozen-lockfile --no-progress --non-interactive",
+    "ci:format": "npx yarn@1.22.10 format",
+    "ci:build": "npx yarn@1.22.10 build",
+    "ci:test": "npx yarn@1.22.10 test",
+    "ci:lint": "npx yarn@1.22.10 lint"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ],
+    "env": {
+      "browser": true,
+      "webextensions": true
+    }
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
```

Will add Safari when they support all the APIs we need, so far I don't believe it's the case. 
```suggestion
      "last 1 chrome version",
      "last 1 firefox version"
```